src/index.html

```diff
@@ -10,13 +10,29 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Moyo header!!</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="img" href="index.html"><img class="img" src="images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <a class="nav_link link1 is_active" href="index.html">apple</a></li>
+        <a class="nav_link link2" href="index.html">samsung</a></li>
+        <a class="nav_link link3" href="index.html">smartphones</a></li>
+        <a class="nav_link link4" href="index.html">laptops&computers</a></li>
+        <a class="nav_link link5" href="index.html">gadgets</a></li>
+        <a class="nav_link link6" href="index.html">tablets</a></li>
+        <a class="nav_link link7" href="index.html">photo</a></li>
+        <a class="nav_link link8" href="index.html">video</a></li>
```

Comment:
Each navigation link is followed by a closing `</li>` tag, but there is no corresponding opening `<li>` tag. If these are meant to be list items, they should be enclosed within `<ul>` or `<ol>` tags, and each link should be wrapped in `<li>` tags.